hack/check_tlsprofile.sh

```diff
@@ -21,26 +21,36 @@
 set -ex
 
 clean_nmap_output () {
-  sed -i '/^$/d' $1
-  sed -i '/^Starting Nmap.*$/d' $1
-  sed -i '/^Nmap scan report for.*$/d' $1
-  sed -i '/^Starting Nmap.*$/d' $1
-  sed -i '/^Host is up.*$/d' $1
-  sed -i '/^Nmap done:.*$/d' $1
-  sed -i '/^MAC Address: .* (Unknown)$/d' $1
+  sed -i '/^$/d' "$1"
+  sed -i '/^Starting Nmap.*$/d' "$1"
+  sed -i '/^Nmap scan report for.*$/d' "$1"
+  sed -i '/^Starting Nmap.*$/d' "$1"
+  sed -i '/^Host is up.*$/d' "$1"
+  sed -i '/^Nmap done:.*$/d' "$1"
+  sed -i '/^MAC Address: .* (Unknown)$/d' "$1"
 }
 
 run_nmap () {
-  nmap -T2 --max-parallelism 1 --max-retries 5 --script +ssl-enum-ciphers -p 4343 ${IPADDR} > $1
+  nmap -T2 --max-parallelism 1 --max-retries 5 --script +ssl-enum-ciphers -p 4343 "${IPADDR}" > "$1"
 }
 
-if ${KUBECTL_BINARY} get service -n ${INSTALLED_NAMESPACE} hyperconverged-cluster-webhook-service ; then
+check_ssp_up() {
+  if [[ -n $SSP_DEPLOYED ]]; then
+    ${KUBECTL_BINARY} wait deployment -n "${INSTALLED_NAMESPACE}" ssp-operator --for=condition=Available --timeout="240s"
+  fi
+}
+
+if [[ $(${KUBECTL_BINARY} get ssp -n "${INSTALLED_NAMESPACE}") ]]; then
+  SSP_DEPLOYED=true
+fi
+
+if ${KUBECTL_BINARY} get service -n "${INSTALLED_NAMESPACE}" hyperconverged-cluster-webhook-service ; then
```

Comment:
i think you can get the webhook service with:
```
${KUBECTL_BINARY} get service -n "${INSTALLED_NAMESPACE}" -l name=hyperconverged-cluster-webhook
```
and then you won't need to find with 2 name possibilities.